guacamole-ext/src/main/java/org/apache/guacamole/net/auth/TokenInjectingUserContext.java

```diff
@@ -100,8 +105,13 @@ public TokenInjectingUserContext(UserContext userContext) {
      * @return
      *     The tokens which should be added to the in-progress call to
      *     connect().
+     *
+     * @throws GuacamoleException
+     *     If the tokens applicable to the given connection group cannot be
+     *     generated.
```

Comment:
Same as above comment for adding this to an existing method...

Reply:
(See https://github.com/apache/guacamole-client/pull/336#discussion_r771785428)